### ISSUE TITLE
Cannot read property 'properties' of undefined when try to install theme

### ISSUE THREAD

**Issue by amirmeimari** [vercel/hyper#3670]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10
- **Hyper.app version**: 3.0.0 (stable) 
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/AmirMeimari/3e9d664a9a301a1d12d8fc7d5e2cef18
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla

## Issue

I'm using hyper with git bash. when i try to install any theme, i get this error
```
C:\Users\amirm\AppData\Local\hyper\app-3.0.0\resources\bin\cli.js:19153
      getProperties = memoize(() => getParsedFile().program.body[0].expression.right.properties),

TypeError: Cannot read property 'properties' of undefined
```

what should i do?

![Annotation 2019-05-14 231447](https://user-images.githubusercontent.com/34706947/57723550-19b95f00-769e-11e9-9a4f-bd5cb088034b.png)

**Comment by inbredtom** [comment]
I also have this issue with hyper-spotify, hyper-opacity

**Comment by colepeters** [comment]
Same issue here on macOS, FWIW.

**Comment by AmirMeimari** [comment]
I've fixed this issue in my windows machine. Somehow i accidentally delete my npm path in `Environment variables` -> `User variables` -> path.
check user path for this: `C:\Users\<yourComputeName>\AppData\Roaming\npm`, if it's not exist, simply add it.

**Comment by colepeters** [comment]
I'm still having this issue on macOS, specifically when trying to install [`xi-hyper`](https://hyper.is/plugins/xi-hyper). Any updates?

**Comment by vivekrp** [comment]
Any temporary fix for this? I tried a few things but doesn't work, also it's not only hyper install doesn't but plugins don't work at all even after adding to .hyper.js.

**Comment by colepeters** [comment]
I've tried everything recommended in similar issues — deleting my hyper config, deleting `/usr/local/bin/hyper`, then doing a complete reinstall, using the default hyper config… nothing has worked for me so far. Has anyone managed to resolve this? Even running `hyper list` results in the same error.

Specs:

Hyper: 3.0.2
macOS 10.15.3

**Comment by colepeters** [comment]
For what it's worth, I've just tried [canary 4 for 3.1.0](https://github.com/zeit/hyper/releases/tag/v3.1.0-canary.4) and this seems to be resolved. However, I noticed when trying to install a plugin, I got a warning that the config file wasn't available:

![image](https://user-images.githubusercontent.com/1713932/75815270-bbf32200-5d58-11ea-8319-9e290fcecfb8.png)

I realized the path for this config file (`~/.config/hyper/.hyper.js`) is different from what I'm used to, that being `~/.hyper.js`.

It was about this time that I noticed the documentation on the Hyper website lists a totally different config location for the current version of Hyper on Mac:

![image](https://user-images.githubusercontent.com/1713932/75815358-e1802b80-5d58-11ea-8aca-0273698f487b.png)

The thing is, at least for Hyper 3.0.2, it seems to automatically create `~/.hyper.js` if it's not already present — and if you remove `~/hyper.js`, it just creates another one. I tried also creating a config file in the specified `~/Library/Application Support/Hyper/.hyper.js`, but this didn't seem to help.

So now I'm all kinds of extra confused 😅
